src/lib/Auth.svelte

```diff
@@ -23,6 +25,8 @@
       if (error) throw error
     } catch (error) {
       console.error(error)
+      console.log('This is the error', error)
```

Comment:
Do we really need to log the error to console twice here?

Reply:
Good catch, I thought I had removed that.  We shouldn't ever want any `console.log`'s typically in production.

Reply:
Pushed up the change to remove it.